### PR TITLE
Update application header

### PR DESCRIPTION
**Pull request by kyle1morel**
# Description

New styling. Popup menu.

https://apps.nrs.gov.bc.ca/int/jira/browse/PADS-361

## Types of changes

New feature (non-breaking change which adds functionality)

## Checklist

- [ ] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have checked that unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: frontend/src/components/layout/Header.vue [hunk 1]
```diff
@@ -1,40 +1,40 @@
...
+
+// Actions
+const { t } = useI18n();
 </script>
 
 <template>
   <header>
     <nav id="header-branding">
-      <div class="flex flex-row flex-wrap align-items-center p-2 lg:pl-6">
+      <div class="flex flex-row flex-wrap align-items-center p-2 pb-0 lg:pl-6">
```
Comment (jujaga) [inline frontend/src/components/layout/Header.vue:13]: It looks like the header height is noticeably larger than the GCPE style bars we see on the official site and Design Guide. Best I can find after some scanning is a `min-height: 65px` implicitly from their documentation example, but wasn't able to derive or find a canonical height measurement for the header element.
Ref: <https://designsystem.gov.bc.ca/react-components/?path=/docs/components-header-header--docs#controls>

I am wondering if the p-2 and pb-0 classes are causing unintended side effects on the bar height. That being said, as there doesn't seem to be a formal prescription on what a header height should precisely be, @Subin1Doo would you be able to make a determination on what the PCNS' header height should be?

**Review by jujaga — COMMENTED** [review]
Looks nice! A few specific questions on precise representation of the header though.

File: frontend/src/components/layout/Header.vue (hunk 1 above)
Comment (Subin1Doo) [reply frontend/src/components/layout/Header.vue:13]: it's set to 54px in the Figma's design guide document

Comment (kyle1morel) [reply frontend/src/components/layout/Header.vue]: There doesn't seem to be a set standard. https://www2.gov.bc.ca/gov/content/home uses 60px so I'm going to conform to that.

**Review by jujaga — APPROVED** [review]

**Review by wilwong89 — APPROVED** [review]

**Pull request merged**